apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -128,16 +136,13 @@ const TutorialDetail = React.createClass({
                 className="modal-body"
                 style={styles.tutorialDetailModalBody}
               >
-                <div className="col-50">
+                <div style={styles.tutorialDetailImageContainer} className="col-xs-12 col-sm-6">
```

Comment:
The mash-up is certainly not ideal, but I wanted tutorialExplorer to take over responsive layout for all its regular UI.  However, we're using Bootstrap's modal dialog, so I'm using entirely Bootstrap for that, which is contained in tutorialDetail.jsx only.